### DISCUSSION TITLE
Docs: Add info about how to split up views

### DISCUSSION THREAD

**Discussion by fkromer** [vitalik/django-ninja#681] [Ideas]
The docs do not contain info about how to [breaking up django-ninja views](https://medium.com/@fwiles/breaking-up-django-ninja-views-9de1f33fccd7). That's needed e.g. if you want to consolidate the APIs of separate Django apps into an API for the overall Django project.

The solution suggested (exemplary Django app `core`) is:

```
# File: core/api.py
from ninja import NinjaAPI

api = NinjaAPI(csrf=True)
```

```
# File: users/views.py
from core.api import api

@api.get("users/something/"):
def get_user(request):
   # ... do something interesting here ...

# File: posts/views.py
from core.api import api

@api.get("posts/{username}/"):
def get_posts(requests, username: str):
   # ... do something interesting here ...
```

Import `views.py` into `urls.py` before calling `api.urls` in `urlpatterns`:

```
# File: config/urls.py
import users.views
import posts.views
from core.api import api

urlpatterns = [
    path("api/", api.urls),
]
```

**Comment by Jragon** [comment]
I think the neater way is to use nested routers right?

We have a central app for configuring our api and setting up all of the error handlers, and auth. It's also a place to store any api utils. `core/api`. 

Then any app has an api folder, which contains a python file for each set of concerns. These all define routers, then in the __init__.py just import all the routers from this app and make one app wide one. Then in the `core/api/api.py` file import all of the app routers and make one router to join them all. Then in urls.py import from `core/api/api.py`

Here's the docs: https://django-ninja.rest-framework.com/guides/routers/?h=nested+routers#nested-routers
